src/runtime-rs/crates/hypervisor/src/qemu/mod.rs

```diff
@@ -0,0 +1,134 @@
+
+mod inner;
+
+use inner::QemuInner;
+use crate::hypervisor_persist::HypervisorState;
+use crate::device::Device;
+use crate::Hypervisor;
+use crate::{VcpuThreadIds, HypervisorConfig};
+use kata_types::capabilities::Capabilities;
+
+use anyhow::{Context, Result};
+use async_trait::async_trait;
+
+use std::sync::Arc;
+use tokio::sync::RwLock;
+
+
```

Comment:
Maybe we need a hint on this commit that this is still a framework and dummy implementation like 👇🏻
```Rust
// Currently WIP
```